introspection-engine/introspection-engine-tests/tests/cockroachdb/gin.rs

```diff
@@ -0,0 +1,102 @@
+use expect_test::expect;
+use introspection_engine_tests::test_api::*;
+use quaint::prelude::Queryable;
+use test_macros::test_connector;
+
+#[test_connector(tags(CockroachDb), preview_features("cockroachDb"))]
+async fn gin_preview_disabled(api: &TestApi) -> TestResult {
+    let schema_name = api.schema_name();
+    let create_table = format!("CREATE TABLE \"{schema_name}\".\"A\" (id SERIAL PRIMARY KEY, data int[] not null)",);
+    let create_idx = format!("CREATE INDEX \"A_data_idx\" ON \"{schema_name}\".\"A\" USING GIN (data);",);
+
+    api.database().raw_cmd(&create_table).await?;
+    api.database().raw_cmd(&create_idx).await?;
```

Comment:
The two statements could go into a single `raw_cmd()` call — that would save some compile and runtime, and make the test generally simpler.